aldryn_newsblog/views.py

```diff
@@ -21,7 +21,7 @@ class ArticleDetail(TranslatableSlugMixin, AppConfigMixin, DetailView):
     slug_field = 'slug'
 
     def get_queryset(self):
-        return Article.objects.active_translations(
+        return Article.published.active_translations(
             translation.get_language()
         ).filter(
             app_config__namespace=self.namespace
```

Comment:
We have QuerySet and Manager in `aldryn-apphooks-config` with `namespace(namespace)` method, so you can use like that:

```python
Article.objects.published().active_translations(translation.get_language()).namespace(self.namespace)
```
They are documented in apphooks docs: https://github.com/aldryn/aldryn-apphooks-config#implementation-step-guide